roles/lib_utils/filter_plugins/oo_filters.py

```diff
@@ -660,6 +660,41 @@ def map_from_pairs(source, delim="="):
     return dict(item.split(delim) for item in source.split(","))
 
 
+def lib_utils_oo_get_node_labels(source, hostvars=None):
+    ''' Return a list of labels assigned to schedulable nodes '''
+    labels = list()
+
+    # Filter out the unschedulable nodes
+    for host in source:
+        if host not in hostvars:
+            return
+        node_vars = hostvars[host]
+        if 'openshift_schedulable' in node_vars and \
```

Comment:
It seems one of your commits reverted this from 'openshift.node.schedulable'

Reply:
Right, turns out this fact doesn't get set anymore. It makes sense though, as by default all nodes (including master) are now schedulable, except ones marked explicitly with `openshift_schedulable=false`.

I guess this assumption is not a very good idea - I'll re-add `openshift.node.schedulable` fact as it currently has got lost

Reply:
Done, now `openshift_facts` would set it, `true` if not set by `openshift_schedulable`, PTAL